db/models.py

```diff
@@ -1,3 +1,5 @@
+from django.contrib.auth.models import User, AbstractUser
```

Comment:
The `User` model is imported but not used in this file. It is a good practice to remove unused imports, as they can lead to confusion and make the code harder to read. Additionally, use `get_user_model()` instead of `AbstractUser` or `User` when you need to refer to the user model. This is a best practice in Django and allows for easier customization of the user model in the future.